docs/alternative-environment-tips/docksal.md

```diff
@@ -0,0 +1,126 @@
+# Setting up BLT with Docksal
+
+By default, BLT is set to work with Drupal VM. It is easy to set up to use with Docksal, but
+you will have to install the Docksal `blt` addon command and make some configuration changes.
+
+## Setting Your Project to Use Docksal
```

Comment:
Maybe substitute in `fin config generate --stack=acquia` here. This will set up the project with memcache and solr as well

Reply:
Good call, makes sense given what we're integrating with...

Reply:
Added with the preferred `fin config set DOCKSAL_STACK=acquia`.